mod/Shattered World - Every state a country/.metadata/metadata.json

```diff
@@ -1,31 +1,12 @@
 {
   "name" : "Shattered World - Every state a country",
   "id" : "",
-  "version" : "",
+  "version" : "1.2.4",
   "supported_game_version" : "",
   "short_description" : "",
   "tags" : [],
   "relationships" : [],
   "game_custom_data" : {
-	"replace_paths" : [
-		"common/history/countries",
-		"common/history/population",
-		"common/history/characters",	
-		"common/history/buildings",			
-		"common/history/diplomacy",			
-		"common/history/diplomatic_plays",
-		"common/history/pops",			
-		"common/history/interests",			
-		"common/history/production_methods",				
-		"common/history/ai",				
-		"common/history/trade_routes",				
-		"common/country_definitions",
-		"common/country_formation",		
-		"common/country_creation",		
-		"common/character_templates",			
-		"common/dynamic_country_map_colors",
-		"common/dynamic_country_names/"
-	],
     "multiplayer_synchronized" : true
```

Comment:
I had an issue with having it in the metadata, it did not work for me. 
But in case you are anyway having all these files in the mod folder, then its no need to list them here. Why do you need it then? I just deleted it and worked. Do I miss something?

Reply:
For example with the country files. The mod adds like 600 new countries but at the same time the game has like the 300 old country files. If you dont exclude them it will still try to create those 300 old countries. While it will work just fine it will throw a lot of errors in the in the debugger.
Ah since you introduced formables, using the metadata like that would probably get rid of your formable tags as they are vanilla? I think. That could have been your issue
In that case you would have to fine tune the exlusions or copy over the formables from vanilla.
Anyhow maybe I have time today or tomorrow to take a proper look^^

Reply:
ah ok, thanks for the explanation, will try to change it back and solve my formables issue. Never looked into the debug log, but good tip, thanks! 👍

Reply:
I tested again, and I dont see any errors in the debug log, are you sure there is an issue?

Reply:
Ah maybe I should have been more precise, i meant the error log. As is stands there are 25k error lines, among those a lot of those:
[09:48:29][jomini_script_system.cpp:263]: Script system error!
  Error: create_character effect [ Wrong scope for effect: none, expected country ]
  Script location: file: common/history/characters/bel - belgium.txt line: 59
  because it cant create the characters for countries that could not have been created, thats why exclusions like common/history/characters are essential to get a cleaner log to actually see the errors :D
  
  Another thing I noticed is that the static path to the game folder obviously wont work for other users of this code ;)
  
  Regardless, I will approve and propose to merge the code as is. I will clean up a few things and fix the exclusions.
  Sorry that I took this long to actually get to it, I got carried away a bit with the Syncretic Cultures mod^^